ods_ci/tests/Tests/0600__distributed_workloads/0602__training/test-run-tuning-stack-tests.robot

```diff
@@ -27,6 +27,15 @@ Run Training operator ODH test base LoRA use case
     ...     TrainingOperator
     Run Training Operator ODH Core Test    TestPytorchjobWithSFTtrainerLoRa
 
+Run Training operator ODH test base QLoRA use case
+    [Documentation]    Run Go ODH tests for Training operator base QLoRA use case
+    [Tags]  RHOAIENG-13142
```

Comment:
Can you add also tag `Resources-GPU`, to make sure that the test is executed with GPU cluster only?

Reply:
I have added Resources-GPU tag accordingly
Thank you @sutaakar for reviewing !